pkgs/os-specific/linux/cpuset/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv
+, fetchgit
+, python2Packages
+}:
+
+python2Packages.buildPythonApplication rec {
+  pname = "cpuset";
+  version = "1.5.8";
+
+  propagatedBuildInputs = [ ];
+
+  makeFlags = [ "prefix=$(out)" ];
+
+  src = fetchgit {
+    url = "${meta.homepage}.git";
```

Comment:
Since this is a github project, use `fetchFromGitHub` instead.

Reply:
Not sure what's the benefit?